cmd/thanos/downsample.go

```diff
@@ -65,6 +92,8 @@ func runDownsample(
 		}
 	}()
 
+	metrics := newDownsampleMetrics(reg)
```

Comment:
Could we use `extprom.WrapRegistererWithPrefix("thanos_compact_downsample", reg)` here?

Reply:
I think it makes sense but with `thanos_compact` prefix, but not a blocker.